src/components/content/privacy-wrapper.tsx

```diff
@@ -0,0 +1,176 @@
+import {
+  faCubes,
+  faPlayCircle,
+  faSpinner,
+} from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import React, { ReactChild } from 'react'
+import styled from 'styled-components'
+
+import { instanceData } from '@/data/en'
```

Comment:
@elbotho: Could you please add the "right" language here? Not sure how to do that since I didn't find a hook for that or something.